src/Mvc/Mvc.Core/src/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -173,10 +173,10 @@ static MvcCoreLoggerExtensions()
                 1,
                 "Executing action method {ActionName} - Validation state: {ValidationState}");
 
-            _actionMethodExecutingWithArguments = LoggerMessage.Define<string, string[], ModelValidationState>(
-                LogLevel.Information,
-                1,
-                "Executing action method {ActionName} with arguments ({Arguments}) - Validation state: {ValidationState}");
+            _actionMethodExecutingWithArguments = LoggerMessage.Define<string, string[]>(
+                LogLevel.Debug,
+                3,
+                "Executing action method {ActionName} with arguments ({Arguments})");
```

Comment:
What happened to the original log message?

IMO it makes much more sense to log what we did before at info (without the arguments) and then make a separate log statement that shows the arguments.

Reply:
That's still there. See `_actionMethodExecuting`